### ISSUE TITLE
sql error

### ISSUE THREAD

**Issue by Pinkeltje** [conseilgouz/pkg_automsg_j4#5]
Hi Pascal,

When moving an article to trash I get an error:
```
danger
You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'AND `state`
```
When disabling automsg the error is gone.

**Comment by conseilgouz** [comment]
Hi Astrid,
Thank you for telling me about this issue.
I've been able to reproduce it with trashed/archived articles.
This will be fixed in version 4.2.9.
Pascal

**Comment by conseilgouz** [comment]
Version 4.2.9 has been released.
